meteor/sofie-server/data-access/repositories/test/mongo-segment-repository.spec.ts

```diff
@@ -147,23 +157,12 @@ describe(`${MongoSegmentRepository.name}`, () => {
 		})
 	})
 
-	// TODO: Extract to Helper Class in Model layer
-	function createPart(params: { id?: string; name?: string; rank?: number; segmentId?: string }): Part {
-		return new Part({
-			id: params.id ?? testDatabase.getValidObjectIdString('id'),
-			name: params.name ?? 'name' + Math.random(),
-			rank: params.rank ?? Math.random(),
-			segmentId: params.segmentId ?? 'segmentId' + Math.random(),
-		} as PartInterface)
-	}
-
-	// TODO: Extract to Helper Class in Model layer
-	function createSegment(params: { id?: string; name?: string; rundownId?: string }): Segment {
-		return new Segment({
-			id: params.id ?? testDatabase.getValidObjectIdString('id'),
-			name: params.name ?? 'name' + Math.random(),
-			rundownId: params.rundownId ?? 'rundownId' + Math.random(),
-		} as SegmentInterface)
+	function createSegment(mongoSegmentInterface?: Partial<MongoSegment>): MongoSegment {
```

Comment:
You are creating a `MongoSegment`, not a `Segment`, so this method should be named `createMongoSegment`

Reply:
Right, missed that on 3 out of 4 of the test clases. Fixed now.